docs/config-example.md

```diff
@@ -18,9 +18,9 @@ spec:
     - node1
     - node[10:100] # All the nodes in your cluster that serve as the worker nodes.
   controlPlaneEndpoint:
-    internalLoadbalancer: haproxy #Internal loadbalancer for apiservers. [Default: ""]
+    internalLoadbalancer: kibe-vip #Internal loadbalancer for apiservers. [Default: ""]
```

Comment:
1. We did not remove the `haproxy` option. So, I think the example file needs to reflect all options. For example, 
    ```
    internalLoadbalancer: haproxy #Internal loadbalancer for apiservers. Support: haproxy, kube-vip [Default: ""]
    ```